iconservice/inv/container.py

```diff
@@ -129,15 +129,22 @@ def __init__(self, is_migrated: bool):
             IconNetworkValueType.STEP_COSTS: None,
             IconNetworkValueType.MAX_STEP_LIMITS: None,
             IconNetworkValueType.SERVICE_CONFIG: None,
-            IconNetworkValueType.IMPORT_WHITE_LIST: None
+            IconNetworkValueType.IMPORT_WHITE_LIST: None,
+            IconNetworkValueType.IREP: None,
         }
 
     @property
     def is_migrated(self) -> bool:
         return self._is_migrated
 
     def get_by_type(self, type_: 'IconNetworkValueType') -> Any:
-        return self._tx_batch.get(type_, self._icon_network_values[type_]).value
+        value = self._tx_batch.get(type_, None)
+        if value is None:
+            value = self._icon_network_values[type_]
+            if value is None:
+                return None
+
+        return value.value
```

Comment:
How about this?
```
def get_by_type(self, type_: 'IconNetworkValueType') -> Optional[Any]:
        value = self._tx_batch.get(type_, self._icon_network_values[type_])
        if isinstance(value, Value):
            return value.value
        # In case of 'None'
        return value
```

Reply:
Good point. I will change the last line to "return None" and apply